### PR TITLE
Add support for message-only and json log formatting

### PR DESCRIPTION
**Pull request by jamesls**

### CODE REVIEW

File: awscli/customizations/logs/tail.py
Comment (kyleknap) [inline awscli/customizations/logs/tail.py:155]: I think we need to remove the `message-only` option from the choices. Looks like an update left over from the previous version

**Review by stealthycoin — APPROVED** [review]

**Review by kyleknap — APPROVED** [review]
Looks good :ship:

**Comment by ebi-yade** [comment]
Hello, maintainer-san 👋🏽 

I would like to ask if this feature is already available.
I found that version 2.4.1 was released after this PR merged with the v2 branch.  Then I tried it and got the result below: an invalid format.
The same command with the latest version ended up with the same result.

I would appreciate it if you give some reaction.  And if not, also I am planning to create an issue after some weeks.

Thanks,

image

**Pull request merged**
